src/components/DevicesView.jsx

```diff
@@ -8,18 +8,28 @@ import { translate } from 'cozy-ui/react/helpers/i18n'
 import Loading from './Loading'
 import Empty from './Empty'
 
+import DevicesModaleRevokeView from './DevicesModaleRevokeView'
+
 // for the icon, we show a phone if it's a phone, and a laptop in all other cases
 const getDeviceKindClass = kind => kind === 'mobile' ? styles['set-device-phone'] : styles['set-device-laptop']
 
-const DevicesView = ({ t, f, isFetching, devices }) => (
+const DevicesView = ({ t, f, isFetching, devices, openDeviceRevokeModale, deviceToRevoke, onDeviceModaleRevoke, onDeviceModaleRevokeClose, devicePerformRevoke }) => (
   <div className={devicesStyles['devices-view']}>
     <h2>{t('DevicesView.title')}</h2>
     { isFetching && <Loading /> }
     { !isFetching && devices.length === 0 && <Empty emptyType='devices' />}
-    { devices.length > 0 && (
+    { !isFetching && devices.length > 0 && (
```

Comment:
Because I run loading (set `isFetching`to `true`) while the DELETE request is sent on the network. Without that condition, the list of devices is pushed to the bottom of the page for a few seconds, it’s not nice.